docs/ref/utils.txt

```diff
@@ -556,6 +556,36 @@ https://web.archive.org/web/20110718035220/http://diveintomark.org/archives/2004
         z = person.friends         # does not call
         x is z                     # is True
 
+.. class:: classproperty
```

Comment:
`classproperty` -> `classproperty(method=None)`

Reply:
Thank you @felixxm, curious why do we mention `(method=None)`, is this because this class doesn't define any method for user to define/overwrite or is there some other thought process behind this?

Reply:
Because that's the signature of this class, see `classproperty.__init__()`.

Reply:
Thanks, got it. So we mention the signature in docs. This is helpful.

**Quick note,** I wasn't able to see the signature using `classproperty.__init__()`, it worked for me using `inspect.signature()`

```
from django.utils.functional import classproperty
print(classproperty.__init__())

---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-8-871ae726864b> in <module>
      2 obj = classproperty()
      3 
----> 4 print(classproperty.__init__())

TypeError: __init__() missing 1 required positional argument: 'self'
```

```
from django.utils.functional import classproperty
obj = classproperty()

print(obj.__init__())
None
```

Finally with `inspect.signature`
```
import inspect
print(inspect.signature(classproperty))

(method=None)
```